ProjectGroupA/src/main/java/com/fundation/search/View/FieldsPanel.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  @(#)AssetFile.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.fundation.search.View;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.*;
+
+public class FieldsPanel extends JPanel {
+    private JTextField pathText;
```

Comment:
if I'm not mistaken this should be a JFileChooser instead of a JTextField right?

Reply:
filechooser it was not implemented yet, it will be put next to this field